spring-kafka/src/test/java/org/springframework/kafka/retrytopic/RetryTopicSameContainerFactoryIntegrationTests.java

```diff
@@ -186,10 +194,6 @@ public ConcurrentKafkaListenerContainerFactory<String, String> kafkaListenerCont
 			props.setPollTimeout(50L);
 			props.setIdlePartitionEventInterval(100L);
 			factory.setConsumerFactory(consumerFactory);
-			DefaultErrorHandler errorHandler = new DefaultErrorHandler(
-					(cr, ex) -> latchContainer.countDownLatch2.countDown(),
-					new FixedBackOff(0, 2));
-			factory.setCommonErrorHandler(errorHandler);
```

Comment:
This is the part we no longer cover anymore.

I just pushed another commit that creates a new bean and keeps `BasicKafkaListener` instead of modifying it: 

https://github.com/spring-projects/spring-kafka/pull/2555/commits/01f0d6d7903ab2294cff0846c4bee86c6d93fe31